dts/arm/nordic/nrf51822.dtsi

```diff
@@ -133,6 +134,18 @@
 			label = "SPI_1";
 		};
 
+		/* nRF TEMP Peripheral:
+		 * .. Default enabled,
+		 * .. Interrupt priority:
+		 * .. .. Range: 0-1, Default: 1
+		 */
```

Comment:
Added these comments in the DTSI file; not sure if this is the practice that we want to have, or if this comment (interrupt range) should go somewhere else? Can the YAML description enforce this restriction?

Reply:
True - i'll remove it.